products/mariadb.md

```diff
@@ -108,17 +119,18 @@ releases:
     lts: true
     releaseDate: 2023-02-16
     eol: 2028-02-16
+    eoes: 2028-02-16
     latest: "10.11.11"
     latestReleaseDate: 2025-02-04
 
 -   releaseCycle: "10.10"
-    releaseDate: 2022-11-17
+    releaseDate: 2022-11-07
```

Comment:
```suggestion
    releaseDate: 2022-06-23
```
![image](https://github.com/user-attachments/assets/a7fc8fee-ceeb-4b9e-897a-3979ad2aa47a)
source : https://mariadb.com/kb/en/mariadb-10100-release-notes/

Reply:
10.10.0 is an alpha release, see https://mariadb.org/about/#maintenance-policy for ga dates.